VotingApplication/VotingApplication.Data/Migrations/Configuration.cs

```diff
@@ -8,7 +8,7 @@ public sealed class Configuration : DbMigrationsConfiguration<VotingContext>
         public Configuration()
         {
             AutomaticMigrationsEnabled = false;
-            AutomaticMigrationDataLossAllowed = false;
+            AutomaticMigrationDataLossAllowed = true;
```

Comment:
This is something we want to avoid on Live without an explicit migration doing the data loss.